lib/rex/powershell/payload.rb

```diff
@@ -40,7 +40,7 @@ def self.to_win32pe_psh(template_path, code)
     hash_sub[:var_iter] 		= Rex::Text.rand_text_alpha(rand(8)+8)
     hash_sub[:var_syscode] 		= Rex::Text.rand_text_alpha(rand(8)+8)
 
-    hash_sub[:shellcode] = Rex::Powershell.to_powershell(code, hash_sub[:var_code])
+    hash_sub[:shellcode] = Rex::Text.to_powershell(code, hash_sub[:var_code])
```

Comment:
the diff reverted this change which is necessary, can you please put this back to Rex::Powershell.to_powershell thanks